src/index.html

```diff
@@ -9,10 +9,61 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.scss"
```

Comment:
Stylesheets with a .scss extension can't be linked directly in HTML. You should compile them to .css files first, then link the compiled .css file here.